scripts/lib/libvirt/fixtures/cloud-admin.netv6.xml

```diff
@@ -0,0 +1,12 @@
+<network>
+  <name>cloud-admin</name>
+  <bridge name='cloudbr' stp='off' delay='0' />
+  <mac address='52:54:00:AB:B1:77'/>
+  <ip family='ipv6' address='fd00::1' prefix='112'>
```

Comment:
fd00::1 is a valid address, which is cool, the examples in the spec were: fc00:0:0:1::/112 which would be: fc00:0:0:1::1. Though maybe we should be real programmers and do zero based counting and actually make the admin/management network fd00:: it would be _much_ easier to write.

Reply:
It's only used in the test code to match the generated XML against this file. I just picked up an address, but I can change it to something bonkers like ffff::